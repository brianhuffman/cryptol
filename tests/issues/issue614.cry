f : {m, n, k} (n == max 2 m k == m + 1) => [m] -> [k][n]
f x = zero


g : {n, k} (0 < n <= k) => [n] -> [k]
g x = sext x
