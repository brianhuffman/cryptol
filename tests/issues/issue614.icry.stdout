Loading module Cryptol
Loading module Cryptol
Loading module Main

[error] at issue614.cry:1:18--1:20 and issue614.cry:1:31--1:33
    The fixities of
      • (==) (precedence 20, non-associative)
      • (==) (precedence 20, non-associative)
    are not compatible.
    You may use explicit parentheses to disambiguate.
[error] at issue614.cry:5:15--5:16 and issue614.cry:5:19--5:21
    The fixities of
      • (<) (precedence 30, non-associative)
      • (<=) (precedence 30, non-associative)
    are not compatible.
    You may use explicit parentheses to disambiguate.
